lib/dialects/mssql/schema/mssql-columncompiler.js

```diff
@@ -63,6 +65,36 @@ class ColumnCompiler_MSSQL extends ColumnCompiler {
     return '';
   }
 
+  defaultTo(value, constraintName) {
```

Comment:
Would be better to add second parameter to all dialects (or at least major ones, such as MySQL and PostgreSQL), so that it works consistently.

Reply:
Do the other dialects name constraints in a similar way? I'd be at a bit of a loss to add it other than in the signature definition of the function

Reply:
I did a quick check, looks like DEFAULT constraint name is pretty much a MSSQL thing, neither MySQL nor PostgreSQL seem to have it. From this perspective it seems like it would be better to have second parameter as an options object with named properties, `constraintName` being supported only for MSSQL. Otherwise when we'll need to add one more option, it'll be an issue.

Reply:
Good point, I've updated the signature to expect an object as the second parameter